bridgesample/src/main/AndroidManifest.xml

```diff
@@ -30,12 +30,50 @@
 
         <activity android:name=".scenario.activity.LargeDataActivity" />
 
+        <activity android:name=".scenario.activity.DeeplinkLargeDataActivity"
+            android:launchMode="singleTask">
+            <intent-filter>
+                <action android:name="android.intent.action.VIEW" />
+                <category android:name="android.intent.category.DEFAULT" />
+                <category android:name="android.intent.category.BROWSABLE" />
+
+                <data
+                    android:pathPattern="/large-activity"
+                    android:host="www.android-bridge.com"
+                    android:scheme="http" />
```

Comment:
Would it be a little cleaner to use a custom scheme? Something like `"bridgesample"` or `"livefrontbridgesample"`? You could also make the `host` just `large-data-activity` at that point, right? Same with below? If there's a good argument to keep the `http` scheme, though, I'm open to it.

Reply:
Hmm...so I was suggesting something more like this:

```xml
                <data
                    android:host="success-activity"
                    android:scheme="bridgesample" />
```

Then the deep links would look like `bridgesample://success-activity` etc. It's just a bit more scoped to the sample app and wouldn't result in the browser accidentally being launched in certain cases.